coderd/database/queries/prebuilds.sql

```diff
@@ -106,41 +107,14 @@ RETURNING *;
 SELECT
     t.name as template_name,
     tvp.name as preset_name,
-		COUNT(*) FILTER ( -- created
-				-- TODO (sasswart): double check which job statuses should be included here
-				WHERE
-					pj.initiator_id = 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-					AND pj.job_status = 'succeeded'::provisioner_job_status
-		) as created,
-		COUNT(*) FILTER ( -- failed
-				-- TODO (sasswart): should we count cancelled here?
-				WHERE pj.initiator_id = 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-						AND pj.job_status = 'failed'::provisioner_job_status
-		) as failed,
-		COUNT(*) FILTER ( -- assigned
-				WHERE pj.initiator_id = 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-						AND NOT w.owner_id = 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-		) as assigned,
-		COUNT(*) FILTER ( -- exhausted
-				-- TODO (sasswart): write a filter to count this
-				-- we should be able to count:
-				-- - workspace builds
-				-- - that have a preset id
-				-- - and that preset has prebuilds enabled
-				-- - and the job for the prebuild was initiated by a user other than the prebuilds user
-				WHERE
-					wb.template_version_preset_id IS NOT NULL
-					AND w.owner_id != 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-					AND wb.initiator_id != 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-		) as exhausted,
-		COUNT(*) FILTER ( -- used_preset
-			WHERE wb.template_version_preset_id IS NOT NULL
-		) as used_preset
-FROM workspace_builds wb
-INNER JOIN provisioner_jobs pj ON wb.job_id = pj.id
-LEFT JOIN workspaces w ON wb.workspace_id = w.id
-LEFT JOIN template_version_presets tvp ON wb.template_version_preset_id = tvp.id
-LEFT JOIN template_versions tv ON tv.id = wb.template_version_id
-LEFT JOIN templates t ON t.id = tv.template_id
-WHERE pj.initiator_id = 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid
-GROUP BY t.name, tvp.name;
+    COUNT(*) as created_count,
+    COUNT(*) FILTER (WHERE pj.job_status = 'failed'::provisioner_job_status) as failed_count,
+    COUNT(*) FILTER (WHERE w.owner_id != 'c42fdf75-3097-471c-8c33-fb52454d81c0'::uuid) as claimed_count
```

Comment:
I understand that this is a special user `prebuilds`? I'm not convinced about hardcoding it in the SQL query. Do you think we can pass from the backend?

Reply:
To clarify: Is this an instruction or are you asking for my opinion?

Reply:
Always opinion, you're the owner of this code.

PS. I think Cian has also raised question about hardcoding ID.